### PR TITLE
Added go runtime identifier

### PR DESCRIPTION
**Pull request by sacOO7**

### CODE REVIEW

File: protocol/agents.json [hunk 1]
```diff
@@ -234,6 +234,14 @@
                 "ably-go"
             ],
             "type": "sdk"
+        },
+        {
+            "identifier": "go",
+            "versioned": true,
+            "emitters": [
+              "ably-go"
```
Comment (lmars) [inline protocol/agents.json:242]: Please use a 4-space indent

Comment (sacOO7) [reply protocol/agents.json:242]: I don't have indentation tool : (
Seems need to do it manually

Comment (sacOO7) [reply protocol/agents.json]: Fixed !

**Review by lmars — APPROVED** [review]
Thanks!

**Pull request merged**